benchmarks/query_benchmarks/query_raw_deferred/benchmark.py

```diff
@@ -0,0 +1,16 @@
+from ...utils import bench_setup
+from .models import MultiField
+
+
+class QueryRawDeferred:
+    def setup(self):
+        bench_setup(migrate=True)
+        for i in range(0, 1000):
+            kwargs = {f"field{j}": f"foobar_{i}_{j}" for j in range(1, 11)}
+            MultiField(**kwargs).save()
+
+    def teardown(self):
+        MultiField.objects.all().delete()
+
+    def time_query_raw_deferred(self):
+        list(MultiField.objects.raw("select id from query_raw_deferred_multifield"))
```

Comment:
What is the difference between this benchmark and `time_query_raw`?

Reply:
The `query_raw_deferred` benchmark only queries a single field while `query_raw` queries all the field, if you feel that the benchmarks are redundant I will remove one of them

Reply:
If they are different that's great, I couldn't see where though. Could you explain where the difference you mentioned is in this PR?

I really don't know much at all about the ORM so I'm interested to learn!

Reply:
I further looked into this the only difference between these two is that the `MultiField` object returned by `list(MultiField.objects.raw("select id from query_raw_deferred_multifield"))` does not contain the data in the other fields but when they are accessed(`Multifield.field1`)  a call is made to extract them. I also checked if they contain the same values by using the `==` operator and the result was `True`. So I guess I was wrong. Should I remove one of them?

Reply:
Ah OK so it sounds like you are seeing the deferall taking place let's have both.